### PR TITLE
Friendlier warning on UCR reports when table has not yet been built.

### PR DESCRIPTION
**Pull request by NoahCarnahan**
cc @nickpell

### CODE REVIEW

File: corehq/apps/userreports/reports/data_source.py [hunk 1]
```diff
@@ -5,7 +5,8 @@
 )
 from sqlalchemy.exc import ProgrammingError
 from corehq.apps.reports.sqlreport import SqlData
-from corehq.apps.userreports.exceptions import UserReportsError
+from corehq.apps.userreports.exceptions import UserReportsError, \
```
Comment (dimagimon) [inline corehq/apps/userreports/reports/data_source.py]: F401 'UserReportsWarning' imported but unused

File: corehq/apps/userreports/reports/view.py [hunk 2]
```diff
@@ -165,6 +167,16 @@ def get_ajax(self, request, domain=None, **kwargs):
             return self.render_json_response({
                 'error': e.message,
             })
+        except TableNotFoundWarning as w:
```
Comment (dimagimon) [inline corehq/apps/userreports/reports/view.py]: F841 local variable 'w' is assigned to but never used

**Comment by nickpell** [comment]
screenshot would be nice, but not blocking

**Comment by NoahCarnahan** [comment]
The button overlap isn't great, but that's an existing problem with the "Edit Report" button and warnings/errors on that page.

screen shot 2015-06-11 at 10 40 17 am

File: corehq/apps/reports_core/templates/reports_core/base_template_new.html [hunk 3]
```diff
@@ -121,6 +127,10 @@
     {% endblocktrans %}
     <div id="error-message"></div>
 </div>
+<div id="report-warning" class="alert alert-warn hide">
+    Warning:
```
Comment (czue) [inline corehq/apps/reports_core/templates/reports_core/base_template_new.html]: `trans`?

File: corehq/apps/userreports/reports/view.py [hunk 4]
```diff
@@ -165,6 +167,16 @@ def get_ajax(self, request, domain=None, **kwargs):
             return self.render_json_response({
                 'error': e.message,
             })
+        except TableNotFoundWarning:
+            if self.spec.report_meta.created_by_builder:
+                msg = "The database table backing your report does not exist yet. " \
+                      "Please wait while the report is populated."
```
Comment (czue) [inline corehq/apps/userreports/reports/view.py]: translate?

File: corehq/apps/reports_core/templates/reports_core/base_template_new.html (hunk 3 above)
Comment (NoahCarnahan) [inline corehq/apps/reports_core/templates/reports_core/base_template_new.html]: woops. Fixed here: https://github.com/dimagi/commcare-hq/pull/7061

**Pull request merged**